src/app/modules/applications/components/servable-names/servable-names.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ApplicationsFacade } from '@app/core/facades/applications.facade';
+import { ServablesFacade } from '@app/core/facades/servables.facade';
+import {
+  Application,
+  Servable,
+  ModelVersion,
+} from '@app/core/data/types';
+
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'hs-servable-names',
+  templateUrl: './servable-names.component.html',
+  styleUrls: ['./servable-names.component.scss']
```

Comment:
For "dumb" components  set  ChangeDetectionStrategy as OnPush, and it will depend only on Input's changes 
https://angular.io/api/core/ChangeDetectionStrategy